pyccl/halos/massdef.py

```diff
@@ -259,7 +207,7 @@ def from_name(cls, name):
                 :math:`\\Delta=200` times the matter density).
 
         Returns:
-            :class:`MassDef` subclass corresponding to the input name.
+            :class:`MassDef` object corresponding to the input name.
```

Comment:
what's the difference?

Reply:
Everything is an object in python. A instantiated class is an instance (https://stackoverflow.com/a/61716167)

Reply:
Wow, such an unenlightening discussion on semantics!
![image](https://github.com/LSSTDESC/CCL/assets/8809493/835930f9-df3a-4b61-996b-154d44f056f2)

> Everything is an object in python

Yes, because everything is an instance of a class (even classes are objects). So instance == object.

Whatever, changed to instance (but only under duress).